file.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: false
+
+# first method bubble sort
+
+def bubble_sort(arr)
+  arr.sort!
+end
+
+# second method
+
+def bubble_sort_by(arr)
+  arr.sort! do |x, y|
+    x.length - y.length
+  end
+end
```

Comment:
**[REQUIRED]**
* We Should not use inbuilt `#sort` method.
* Remember to raise an error in no block is given.